src/components/Datatable/Table/temp/IndeterminateCheckbox.tsx

```diff
@@ -0,0 +1,43 @@
+// TODO: move this as next state of the `forms/Checkbox`
+
+import React, { forwardRef, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+
+const useCombinedRefs = (...refs): React.MutableRefObject<HTMLInputElement> => {
```

Comment:
It is a bit tricky part. Please why do we need to combine those references?

Reply:
Don't worry about that, I'm not sure if we need that at all, it copied from some example

Reply:
Okey then

Reply:
@PrincAm provided a link with explanation